### ISSUE TITLE
Builder

### ISSUE THREAD

**Issue by Crt123** [molstar/molstar#255]
Is it possible to add model builder functionality (for small molecules) like ch5m3d has:

https://jcheminf.biomedcentral.com/articles/10.1186/1758-2946-5-46

All the best

**Comment by dsehnal** [comment]
Yes, it is possible. Would you be interested in helping to implement it?
